<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBar</class>
 <widget class="QWidget" name="ToolBar">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>132</width>
    <height>525</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QPushButton" name="button_load">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>load</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_refresh">
     <property name="text">
      <string>refresh</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_clear">
     <property name="text">
      <string>clear</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="list_cloud"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
